### ISSUE TITLE
Dockerfile - SSL Error

### ISSUE THREAD

**Issue by ricaldeira** [bcgov/von-network#109]
Hi,
When trying to build the docker image (./mange build) it was giving an SSL Error.
Easily resolved with this line in Dockerfile, before running requirements.txt:

RUN python -m pip install --trusted-host pypi.python.org --trusted-host files.pythonhosted.org --trusted-host pypi.org --upgrade pip 

VirtualBox with Ubuntu server 18 as guest
Host: Windows 10.

Cheers

**Comment by ianco** [comment]
Hi, it would be great if you could send us this update in a Pull Request :-)

**Comment by ricaldeira** [comment]
ok, I'll do that. Thanks

**Comment by WadeBarnes** [comment]
and reference this issue in the PR.

**Comment by WadeBarnes** [comment]
@ricaldeira, can you provide details on the error itself?

**Comment by WadeBarnes** [comment]
I ask, because I am not seeing an SSL error when building using Docker Desktop on Windows 10.

**Comment by WadeBarnes** [comment]
We do see errors to the affect of "certificate is not yet valid" with some builds in some projects, but that is due to a clock shift issue in Docker.  Restarting Docker resolves that issue.

**Comment by ricaldeira** [comment]
Hi Wade, when I tried to reproduce with a cloned repository it didn't occur anymore. Just when I downloaded the zip file. I'll try to reproduce it and get back to you here. 
Thanks

**Comment by WadeBarnes** [comment]
@ricaldeira, were you ever able to reproduce the issue?  Can you summarize and close this issue?

**Comment by ricaldeira** [comment]
Hi Wade, sorry for the late reply. No I have not reproduced it. Thanks.

**Comment by EtricKombat** [comment]
@WadeBarnes , an year before i faced something similar listing it down 👇   hope this helps. (not sure if it is related as well )

https://github.com/EtricKombat/KOBDevOps/issues/16
